src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1493,12 +1497,14 @@ def troubleshoot(cmd, client, resource_group_name, cluster_name, kube_config=Non
         except Exception as ex:
             tr_logger.error("Error occured while checking if the MSI certificate has expired: {}".format(str(ex)))
 
-        storage_account_name, sas_token, readonly_sas_token = utils.setup_validate_strorage_account(cmd.cli_ctx, storage_account, sas_token, resource_group_name)
+        storage_account_name, sas_token, readonly_sas_token = utils.setup_validate_storage_account(cmd.cli_ctx, storage_account, sas_token, resource_group_name)
         if storage_account_name:  # When validated the storage account
             utils.try_upload_log_file(storage_account_name, sas_token, troubleshoot_log_path)
             # token_in_storage_account_url = readonly_sas_token if readonly_sas_token is not None else sas_token
             # ADD the periscope part here.
 
+            utils.collect_logs(cmd, resource_group_name, cluster_name, storage_account_name, sas_token, 
```

Comment:
I think, we dont use 'cmd' as a parameter in collect_logs. Lets remove it.

Reply:
Rest otherwise things LGTM.

Please test once with AKS cluster as well as with a kind cluster. 
Post which, you can merge the pr.

Reply:
Ok. I tested with AKS cluster. The storage container name created is bugbashcluster-dns-1bbdb8a7. Going to test with kind cluster.

Reply:
When I ran the command with the kind cluster, I got an error from periscope saying, "cannot read kube-config" when it called the APIServerFQDN, so no storage container name was created in this case since the logs weren't able to get exported to the blob storage.  Not sure if this happened to you. I'll try troubleshooting some more tomorrow.